client/src/Coin.tsx

```diff
@@ -0,0 +1,17 @@
+import React, { useState } from "react";
+import {Card} from "@material-ui/core";
+
+function Coin(props: { image: string, name: string, price:number }) {
```

Comment:
Try using the ES6 arrow syntax.

Reply:
```suggestion
const Coin = (props: { image: string, name: string, price:number }) => {
```

Reply:
I'm not sure that I prefer using the `const` syntax here. With `function` you have it be declared as `export default` without having an extra line at the end, and you also avoid issues with hosting as well.